sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_orchestration_app_conv_response_async.py

```diff
@@ -0,0 +1,105 @@
+    In this sample, orchestration project's top intent will map to a conversation project.
+
+    For more info about how to setup a CLU orchestration project, see the README.
+
+USAGE:
+    python sample_analyze_orchestration_app_conv_response_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_CLU_ENDPOINT                       - endpoint for your CLU resource.
+    2) AZURE_CLU_KEY                            - API key for your CLU resource.
+    3) AZURE_CLU_ORCHESTRATION_PROJECT_NAME     - project name for your CLU orchestration project.
+    4) AZURE_CLU_ORCHESTRATION_DEPLOYMENT_NAME  - deployment name for your CLU orchestration project.
+"""
+
+import asyncio
+
+async def sample_analyze_orchestration_app_conv_response_async():
+    # [START analyze_orchestration_app_conv_response_async]
+    # import libraries
+    import os
+    from azure.core.credentials import AzureKeyCredential
+
+    from azure.ai.language.conversations.aio import ConversationAnalysisClient
+    from azure.ai.language.conversations.models import ConversationTargetIntentResult
+
+    # get secrets
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_ORCHESTRATION_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_ORCHESTRATION_DEPLOYMENT_NAME"]
+
+    # analyze query
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
+    async with client:
+        query = "Send an email to Carol about the tomorrow's demo"
+        result = await client.analyze_conversation(
+            body={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
+                        "id": "1",
+                        "modality": "text",
+                        "language": "en",
+                        "text": query
+                    },
+                    "isLoggingEnabled": False
+                },
+                "parameters": {
+                    "projectName": project_name,
+                    "deploymentName": deployment_name,
+                    "verbose": True
+                }
+            }
+        )
+
+        # view result
+        print("query: {}".format(result.results.query))
+        print("project kind: {}\n".format(result.results.prediction.project_kind))
+
+        # top intent
+        top_intent = result.results.prediction.top_intent
+        print("top intent: {}".format(top_intent))
+        top_intent_object = result.results.prediction.intents[top_intent]
+        print("confidence score: {}".format(top_intent_object.confidence))
+        print("project kind: {}".format(top_intent_object.target_kind))
+
+        # conversation result
+        if isinstance(top_intent_object, ConversationTargetIntentResult):
```

Comment:
Our samples should not need to use `isinstance` - which is not good practice in Python.
Is this necessary? What are the other possible values/types for `top_intent_object`?

Reply:
Looks like it could also be `LUISTargetIntentResult`, and possibly others. In this case there should be a `kind` field we can check. Something like:
```python
if top_intent_object.kind == 'ConversationIntentResult:`
```
If not - that's a problem we'll need to get fixed in the REST API :)